README.md

```diff
@@ -18,14 +18,8 @@ of your new JupyterLab extension.
 cookiecutter https://github.com/jupyter/jupyterlab-extension-cookiecutter
 ```
 
-## Install and enable your extension
+A very simple working extension, written in common Javascript, is included in
```

Comment:
As the local documentation/docstring nitpicker at Brookhaven, your nitpicks warm my heart. :- D